dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -727,12 +783,10 @@ else if (field.getFieldType().equals(ESMappingConstants.FIELD_ELASTIC_TYPE_DATE)
 								UtilMethods.listToString(valueObj.toString()));
 					}
 				} else if (field.getFieldType().equals(ESMappingConstants.FIELD_TYPE_KEY_VALUE)){
-					final boolean fileMetadata =
-							field.getVelocityVarName().equals(FileAssetAPI.META_DATA_FIELD)
-									&& structure.getStructureType() == Structure.STRUCTURE_TYPE_FILEASSET;
+					//final boolean fileMetadata = field.getVelocityVarName().equals(FileAssetAPI.META_DATA_FIELD) && structure.getStructureType() == Structure.STRUCTURE_TYPE_FILEASSET;
```

Comment:
why is this line commented?

Reply:
Part of the refactoring. Since now that method only takes care of normal keyValues. The logic to handle metadata was moved down. After the metadata has been generated. I'll get it removed

Reply:
done.